src/views/Home.vue

```diff
@@ -9,10 +11,18 @@
 
 <script lang="ts">
 import { Options, Vue } from "vue-class-component";
-
+import Header from "@/components/Header/Header.vue";
 @Options({
   components: {
+    Header
   }
 })
-export default class Home extends Vue {}
+export default class Home extends Vue {
+  private imageHeader =
+      "https://www.flaticon.com/svg/vstatic/svg/2833/2833438.svg?token=exp=1610442341~hmac=78691034b920a1b5599d482745dddbc9"
```

Comment:
brakuje średnika. Przed commitem dobrze jest zrobić autoformatowanie kodu. W zależnośći jakiego IDE używasz i na jakim systemie pracujesz to się robi trochę inaczej. Poczytaj na necie jak to zrobić na setupie którego Ty używasz :)